packages/components/src/theme/ComposerTheme/Components/InlineAlert.ts

```diff
@@ -0,0 +1,38 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/theme';
+
+const messageStylesByStatus = {
+  success: {
+    textStyle: 'body-meta-regular',
+    color: 'text-success',
+  },
+  error: {
+    textStyle: 'body-meta-regular',
+    color: 'text-error',
+  },
+  warning: {
+    textStyle: 'body-meta-regular',
+    color: 'text-warning',
+  },
+  info: {
+    textStyle: 'body-meta-regular',
+    color: 'text-secondary',
+  },
+};
+
+export const InlineAlert: ComponentMultiStyleConfig = {
+  parts: ['container', 'description', 'icon'],
+  baseStyle: ({ status, ...rest }) => {
+    return {
+      container: {
+        alignItems: 'center',
+        gap: '0.5rem',
+      },
+      message: {
+        ...messageStylesByStatus[status],
+      },
+      icon: {
+        ...rest.theme.components.Alert.baseStyle(rest).icon.status[status],
```

Comment:
I wanted to use somethig like this. as an object:
`...Alert.baseStyle.icon.status[status]`
I din't get anything, due to Alert theme is defined as function in its `baseStyles` property, so I changed it `...Alert.baseStyle(props).icon.status[status]`
but I was getting another error from typescript.